site/app/Resources/views/layout.html.twig

```diff
@@ -22,8 +22,11 @@
   {% if app.request.hasPreviousSession %}
     {% for type, messages in app.session.flashbag.all() %}
       {% for message in messages %}
-        <div class="alert alert-{{ type }}" role="alert">
-          {{ message|raw }}
+        <div class="alert alert-{{ type }} alert-dismissable" role="alert">
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+          {{ message | trans | raw }}
```

Comment:
We don't have translated strings anywhere, just hardcode it here in English.

Reply:
I use some translations as the lines were being too long when included in code and were a bit testing-unfriendly